### DISCUSSION TITLE
Unable to load bundle identifiers

### DISCUSSION THREAD

**Discussion by ZilbershteinMiki** [codemagic-ci-cd/codemagic-docs#2512] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Select code signing method
automatic

### Expected results

success

### Actual results

Unable to load bundle identifiers

Loading bundle identifiers for carmel tennis failed. Authentication credentials are missing or invalid. - Provide a properly configured and signed bearer token, and make sure that it has not expired. Learn more about Generating Tokens for API Requests https://developer.apple.com/go/?id=api-generating-tokens

**Comment by himesh-cm** [comment]
Hey @MikiZilbershtein,
Please confirm if you followed the steps mentioned [here](https://docs.codemagic.io/yaml-publishing/app-store-connect/#creating-the-app-store-connect-api-key) in our documentation.

**Reply by ZilbershteinMiki** [reply]
Anyone can help? I can give more information if needed

**Reply by himesh-cm** [reply]
Hey @MikiZilbershtein, please confirm if you have tried as per my last suggestion.

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
